src/components/TimePicker/pageObject/index.js

```diff
@@ -0,0 +1,237 @@
+const PageModal = require('../../Modal/pageObject');
+
+/**
+ * TimePicker page object class.
+ * @class
+ * @tutorial picklist
+ */
+class PageTimePicker {
+    /**
+     * Create a new TimePicker page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the TimePicker root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+        this.timePickerModal = '#time-picker_modal';
```

Comment:
since this is a constant that does not depend on nothing form class we can put it outside
also here you need to create the PageModal only one time